packages/core/helper-plugin/lib/src/hooks/useAPIErrorHandler/utils/formatAxiosError/index.js

```diff
@@ -0,0 +1,13 @@
+import { getPrefixedId } from '../getPrefixedId';
+
+export function formatAxiosError(error, { intlMessagePrefixCallback, formatMessage }) {
```

Comment:
what advantages do you see this being in it's own module?

Reply:
Because all the other utils are one too. If I'd combine everything in one the file would get quite big. This way I can co-locate the tests better.

Reply:
Is it exposed? `import { formatAxiosError } from "@strapi/helper-plugin"`? If it's not it just feels like a shallow module that has almost the same tests as the `useAPIErrorHandler`. I'm not opposed to it though 👍🏼

Reply:
No it is not. It is a private module of the hook. Where would you put the module + the tests?

Reply:
IMO I would have just written one big file `useAPIErrorHandler` and have all it's utils there and test the public facing API gives me what I want and does what I expect it to that would be a deep module sure, but then it has a small API surface to know how to use it and even better theres no cognitive overhead of "where is this function, is it used in another file".

I'm not saying it has to change by the way.